.github/workflows/changelog.yml

```diff
@@ -1,33 +1,35 @@
-name: CI
+# This workflow will build a golang project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-go
+
+name: changelog
 
 on:
   pull_request:
-    branches:
-      - develop
+    branches: [ "develop" ]
 
 jobs:
-  changed_files:
-    runs-on: ubuntu-latest 
-    name: Check CHANGELOG.md
+  run-changelog-check:
+    runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
-      - name: changelog modified
-        id: changelog-modified
+      - name: Checkout source code
+        uses: actions/checkout@v3
+
+      - name: Download unclog binary
+        uses: actions/download-artifact@v4
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          repository: OffchainLabs/unclog
+          name: unclog
+          run-id: 12679708867
```

Comment:
Discussed offline that the maximum artifact retention period is 90 days so we will need to address this in a follow up.

https://docs.github.com/en/actions/administering-github-actions/usage-limits-billing-and-administration#artifact-and-log-retention-policy